backend/tests/integration/test_cards.py

```diff
@@ -18,6 +19,7 @@ async def test_create_card_returns_card(db_session, user, user_client):
         "id": is_uuid_string(),
         "deck_id": deck_id,
         "deck_name": "deck",
+        "deck_path": ANY,
```

Comment:
Using 'ANY' in these tests because test_decks.py already tests the path building logic, and this just copies that